integrations/tensorflow-serving/documentation.yaml

```diff
@@ -0,0 +1,115 @@
+exporter_type: included
+app_name_short: TF Serving
+app_name: TensorFlow Serving
+app_site_name: {{app_name_short}}
+app_site_url: https://github.com/tensorflow/serving
+exporter_name: {{app_name_short}}
+exporter_pkg_name: {{app_name_short}}
+exporter_repo_url: https://www.tensorflow.org/tfx/serving/serving_config#monitoring_configuration
+additional_prereq_info: |
+  {{app_name_short}} exposes Prometheus-format metrics when the `--monitoring_config_file` flag
+  is used to specify a file containing a MonitoringConfig protocol buffer.
+  
+  This is an example:
```

Comment:
Of what?  Is this the contents of a file that you pass via the "--monitoring_config_file" flag? It sound like we need to say something like

1. Create a file that contains a MonitoringConfig protocol buffer. For example, you might create a file named `monitoring-config.txt that contains the following:

     <pre>prometheus_config {
    enable: true,
    path: "/monitoring/prometheus/metrics"
  } </pre>

1. Set the environment variable <code>PATH_TO_MONITORING_CONFIG</code> to the location of your configuration file.

Reply:
Agree this is underspecified. Let's wait for Chelsey to [respond to my q above](https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/913#discussion_r1949163484) about how she had set this up, and then we can rework this.

Reply:
Resolved; this is incorporated into cl/726303300